src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -6,7 +6,10 @@
 class WPML_Elementor_Translatable_Nodes implements IWPML_Page_Builders_Translatable_Nodes {
 
 	const SETTINGS_FIELD = 'settings';
-	const TYPE           = 'widgetType';
+
+	const TYPE = 'widgetType';
+
+	const DEFAULT_HEADING_TAG = 'h2';
```

Comment:
@Igor-WPML we can save some bytes not using empty lines between declarations of the same type:

```php
const SETTINGS_FIELD      = 'settings';
const TYPE                = 'widgetType';
const DEFAULT_HEADING_TAG = 'h2';
```

Reply:
@andreasciamanna I proceed in this way, because otherwise Ctrl+Alt+L (Reformat Code) in phpStorm formats lines like that

    	const SETTINGS_FIELD = 'settings';
	const TYPE = 'widgetType';
	const DEFAULT_HEADING_TAG = 'h2';

and immediately reports

    phpcs: Generic.Formatting.MultipleStatementAlignment.NotSameWarning: Equals sign not aligned with surrounding assignments; expected 16 spaces but found 1 space

Reply:
@kagg-design just fix your IDE settings then:

![image](https://user-images.githubusercontent.com/181780/53243941-b379fd00-36a9-11e9-8207-8fab96b7d148.png)

![image](https://user-images.githubusercontent.com/181780/53243930-ac52ef00-36a9-11e9-98c4-1d4b47167b2e.png)

Reply:
@andreasciamanna thank you for explanations. I completely forgot about these settings.

Constants now are aligned as proposed.